testsuite/MDAnalysisTests/converters/test_parmed.py

```diff
@@ -187,7 +187,13 @@ def test_equivalent_atoms(self, ref, output):
             for attr in self.almost_equal_atom_attrs:
                 ra = getattr(r, attr)
                 oa = getattr(o, attr)
-                assert_almost_equal(ra, oa, decimal=2, err_msg='atom {} not almost equal for atoms {} and {}'.format(attr, r, o))
+                assert_allclose(
+                    ra,
+                    oa,
+                    rtol=0,
+                    atol=1e-2,
+                    err_msg=('atom {} not almost equal for atoms' +
```

Comment:
I'm not super fond of the formatting given we end up using a whole line for a two character variable. I won't block on that but I would ask to use f-strings and implicit continuation here instead of `+`